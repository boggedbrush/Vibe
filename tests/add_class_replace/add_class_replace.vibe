# VibeSpec: 1.0
patch_type: add_class
file: hello.py
--- code: |
class Greeter:
    def __init__(self, name):
        print(f'Greetings, {name}! (new)')

    def greet(self):
        print("Greeter 2.0 at your service!")
