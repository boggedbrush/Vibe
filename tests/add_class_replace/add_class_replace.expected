# hello.py
def greet(name):
    print(f"Hello, {name}!")

class Greeter:
    def __init__(self, name):
        print(f'Greetings, {name}! (new)')

    def greet(self):
        print("Greeter 2.0 at your service!")

